packages/core/src/Stores/client-store.js

```diff
@@ -728,6 +728,11 @@ export default class ClientStore extends BaseStore {
         return this.isBotAllowed();
     }
 
+    @computed
+    get is_pending_proof_of_ownership() {
+        return this.account_status?.authentication?.needs_verification?.find(a => a === 'ownership') === 'ownership';
```

Comment:
`Array.includes()` suits your need better. Then you dont need to do another comparison to turn it to a boolean